File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int maxi = 0, n=prices.size();
        for (int i=0;i<n-1;i++){
            for(int j=i+1;j<n;j++){
                maxi = max(maxi, prices[j]-prices[i]);
            }
        }
        return maxi;
    }
};

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        ans, i = [], 0
        while(i<=len(nums)-k):
            maxi = -10001
            j = i
            while(j<i+k):
                maxi = max(maxi, nums[j])
                j+=1
            ans.append(maxi)
            i+=1
        return ans
